ompi/mpiext/cuda/configure.m4

```diff
@@ -17,6 +17,10 @@
 AC_DEFUN([OMPI_MPIEXT_cuda_CONFIG],[
     AC_CONFIG_FILES([ompi/mpiext/cuda/Makefile])
     AC_CONFIG_FILES([ompi/mpiext/cuda/c/Makefile])
+    AC_CONFIG_HEADER([ompi/mpiext/cuda/c/mpiext_cuda_c.h])
+
+    AC_DEFINE_UNQUOTED([MPIX_CUDA_AWARE_SUPPORT],$CUDA_SUPPORT,
+                       [Macro that is set to 1 when CUDA-aware support is configured in])
```

Comment:
This is a super-picky note: but you might want to update this comment to say that the value is always 0 or 1 (just to distinguish it from being 1 or undefined).